api.go

```diff
@@ -324,6 +324,28 @@ func (lfm *LineFragmentMatch) sizeBytes() (sz uint64) {
 	return
 }
 
+type FlushReason uint8
+
+const (
+	FlushReasonTimerExpired FlushReason = 1 << iota
```

Comment:
do you plan on combining them? IE did you intend for these to be bit flags. or would an `1 + iota` be more appropriate?